src/components/Calculator.js

```diff
@@ -42,7 +31,187 @@ function App() {
         {' '}
         { ' ' }
         {' '}
-        { input }
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
```

Comment:
- [OPTIONAL] Kindly remove the redundancy in your code, because your code looks messy and unprofessional and whatever the things you used for any specific reason do not reflect the use case. Removing the redundant code will help any person to understand your code in a better way and make your code-base cleaner, beautiful and well-organized.